src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/listeners/DoCleanupListener.groovy

```diff
@@ -29,5 +47,26 @@ class DoCleanupListener extends AbstractSpringListener {
                 }
             }
         }
+        if (ISL_PROPS_DB_RESET in error.method.getAnnotation(Tags).value()) {
+            def isls = topology.getIslsForActiveSwitches().collectMany { [it, it.reversed] }
+            log.info("Resetting ISLs bandwidth due to the failure in " + error.method.parent.name + "\nISLs: " + isls)
+            isls.each { database.resetIslBandwidth(it) }
```

Comment:
Can you add method `database.resetIslsBandwidth(List<Isl>)`? 
It's better to do everything in one DB transaction than sending 20+ consecutive requests to DB.

Reply:
done